<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWFormSetTarget</class>
 <widget class="QWidget" name="QWFormSetTarget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <action name="actImport">
   <property name="text">
    <string>Import Setting</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>Save Setting As</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actResetAll">
   <property name="text">
    <string>Reset All</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actResetLoc">
   <property name="text">
    <string>Reset Current Page</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actSeparator1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </action>
  <action name="actSeparator2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </action>
  <action name="actSeparator3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
